2025-05-18 12:41:04 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 12:41:04 - INFO - INFO - [logger.py:104] - Identified 0 support and 0 resistance levels
2025-05-18 12:42:00 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 12:42:00 - INFO - INFO - [logger.py:104] - Identified 0 support and 0 resistance levels
2025-05-18 12:43:26 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 12:43:26 - INFO - INFO - [logger.py:104] - Identified 0 support and 0 resistance levels
2025-05-18 12:44:23 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 12:44:23 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 13:30:31 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 13:30:31 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 13:39:07 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 13:39:07 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 13:39:07 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: AnalysisIndicators.sma() got multiple values for argument 'length'
2025-05-18 13:40:42 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 13:40:43 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 13:40:43 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: AnalysisIndicators.sma() got multiple values for argument 'length'
2025-05-18 13:57:49 - INFO - INFO - [logger.py:104] - Running full volatility analysis for BTC on 1d timeframe
2025-05-18 13:57:49 - INFO - WARNING - [logger.py:108] - Знайдено нульові або від'ємні значення в OHLC даних - ці точки будуть пропущені
2025-05-18 14:02:23 - INFO - INFO - [logger.py:104] - Running full volatility analysis for BTC on 1d timeframe
2025-05-18 14:02:23 - INFO - WARNING - [logger.py:108] - Знайдено нульові або від'ємні значення в OHLC даних - ці точки будуть пропущені
2025-05-18 14:02:23 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:02:23 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:02:23 - INFO - INFO - [logger.py:104] - Обчислення метрик ризику волатильності
2025-05-18 14:02:23 - INFO - INFO - [logger.py:104] - Виявлено 188 проривів волатильності з 2831 точок
2025-05-18 14:02:23 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 7 elements
2025-05-18 14:02:23 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 12 elements
2025-05-18 14:02:23 - INFO - ERROR - [logger.py:112] - Error in full volatility analysis for BTC: unhashable type: 'Series'
2025-05-18 14:08:08 - INFO - INFO - [logger.py:104] - Running full volatility analysis for BTC on 1d timeframe
2025-05-18 14:08:09 - INFO - WARNING - [logger.py:108] - Знайдено нульові або від'ємні значення в OHLC даних - ці точки будуть пропущені
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Обчислення метрик ризику волатильності
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Виявлено 188 проривів волатильності з 2831 точок
2025-05-18 14:08:09 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 7 elements
2025-05-18 14:08:09 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 12 elements
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:08:09 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Обчислення імпульсного відгуку волатильності з розміром шоку 3
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:08:09 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:08:09 - INFO - WARNING - [logger.py:108] - Не вдалося підігнати GARCH модель для імпульсного відгуку
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Підготовка функцій волатильності для ML з вікнами [7, 14, 30]
2025-05-18 14:08:09 - INFO - INFO - [logger.py:104] - Створення ознак волатильності...
2025-05-18 14:08:09 - INFO - ERROR - [logger.py:112] - Помилка при підготовці функцій волатильності для ML: loop of ufunc does not support argument 0 of type float which has no callable log method
2025-05-18 14:08:09 - INFO - ERROR - [logger.py:112] - Error analyzing crypto market conditions: 'close'
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Running full volatility analysis for BTC on 1d timeframe
2025-05-18 14:09:11 - INFO - WARNING - [logger.py:108] - Знайдено нульові або від'ємні значення в OHLC даних - ці точки будуть пропущені
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Обчислення метрик ризику волатильності
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Виявлено 188 проривів волатильності з 2831 точок
2025-05-18 14:09:11 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 7 elements
2025-05-18 14:09:11 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 12 elements
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:09:11 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Обчислення імпульсного відгуку волатильності з розміром шоку 3
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:09:11 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:09:11 - INFO - WARNING - [logger.py:108] - Не вдалося підігнати GARCH модель для імпульсного відгуку
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Підготовка функцій волатильності для ML з вікнами [7, 14, 30]
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Створення ознак волатильності...
2025-05-18 14:09:11 - INFO - ERROR - [logger.py:112] - Помилка при підготовці функцій волатильності для ML: loop of ufunc does not support argument 0 of type float which has no callable log method
2025-05-18 14:09:11 - INFO - ERROR - [logger.py:112] - Error analyzing crypto market conditions: 'close'
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Аналіз кросс-активної волатильності для 3 активів
2025-05-18 14:09:11 - INFO - INFO - [logger.py:104] - Успішно проаналізовано кросс-активну волатильність
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Running full volatility analysis for BTC on 1d timeframe
2025-05-18 14:10:35 - INFO - WARNING - [logger.py:108] - Знайдено нульові або від'ємні значення в OHLC даних - ці точки будуть пропущені
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Обчислення метрик ризику волатильності
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Виявлено 188 проривів волатильності з 2831 точок
2025-05-18 14:10:35 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 7 elements
2025-05-18 14:10:35 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 12 elements
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:10:35 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Обчислення імпульсного відгуку волатильності з розміром шоку 3
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:10:35 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:10:35 - INFO - WARNING - [logger.py:108] - Не вдалося підігнати GARCH модель для імпульсного відгуку
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Підготовка функцій волатильності для ML з вікнами [7, 14, 30]
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Створення ознак волатильності...
2025-05-18 14:10:35 - INFO - ERROR - [logger.py:112] - Помилка при підготовці функцій волатильності для ML: loop of ufunc does not support argument 0 of type float which has no callable log method
2025-05-18 14:10:35 - INFO - ERROR - [logger.py:112] - Error analyzing crypto market conditions: 'close'
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Аналіз кросс-активної волатильності для 3 активів
2025-05-18 14:10:35 - INFO - INFO - [logger.py:104] - Успішно проаналізовано кросс-активну волатильність
2025-05-18 14:14:26 - INFO - INFO - [logger.py:104] - Running full volatility analysis for BTC on 1d timeframe
2025-05-18 14:14:27 - INFO - WARNING - [logger.py:108] - Знайдено нульові або від'ємні значення в OHLC даних - ці точки будуть пропущені
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Обчислення метрик ризику волатильності
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Виявлено 188 проривів волатильності з 2831 точок
2025-05-18 14:14:27 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 7 elements
2025-05-18 14:14:27 - INFO - ERROR - [logger.py:112] - Помилка при вилученні сезонності з волатильності: Length mismatch: Expected axis has 1 elements, new values have 12 elements
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:14:27 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Обчислення імпульсного відгуку волатильності з розміром шоку 3
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Підгонка GARCH(1,1) моделі
2025-05-18 14:14:27 - INFO - ERROR - [logger.py:112] - Помилка підгонки GARCH моделі: 'Series' object has no attribute 'columns'
2025-05-18 14:14:27 - INFO - WARNING - [logger.py:108] - Не вдалося підігнати GARCH модель для імпульсного відгуку
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Підготовка функцій волатильності для ML з вікнами [7, 14, 30]
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Створення ознак волатильності...
2025-05-18 14:14:27 - INFO - ERROR - [logger.py:112] - Помилка при підготовці функцій волатильності для ML: loop of ufunc does not support argument 0 of type float which has no callable log method
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Визначення фаз ринку за допомогою 4 режимів
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Успішно визначено 4 фази ринку
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Виявлення 3 режимів волатильності за методом kmeans
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Успішно виявлено 3 режимів волатильності
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Аналіз кросс-активної волатильності для 3 активів
2025-05-18 14:14:27 - INFO - INFO - [logger.py:104] - Успішно проаналізовано кросс-активну волатильність
2025-05-18 14:21:34 - INFO - INFO - [logger.py:104] - Running full volatility analysis for ['BTC', 'SOL', 'ETH'] on 1d timeframe
2025-05-18 14:21:34 - INFO - ERROR - [logger.py:112] - Error in full volatility analysis for ['BTC', 'SOL', 'ETH']: 'list' object has no attribute 'upper'
2025-05-18 14:21:54 - INFO - INFO - [logger.py:104] - Running full volatility analysis for ['BTC', 'SOL', 'ETH'] on 1d timeframe
2025-05-18 14:21:54 - INFO - ERROR - [logger.py:112] - Error in full volatility analysis for ['BTC', 'SOL', 'ETH']: 'list' object has no attribute 'upper'
2025-05-18 15:17:41 - INFO - INFO - [logger.py:104] - Running full volatility analysis for ['BTC', 'SOL', 'ETH'] on 1d timeframe
2025-05-18 15:17:41 - INFO - ERROR - [logger.py:112] - Error in full volatility analysis for ['BTC', 'SOL', 'ETH']: 'list' object has no attribute 'upper'
2025-05-18 15:18:10 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 15:18:10 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 15:18:10 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: AnalysisIndicators.sma() got multiple values for argument 'length'
2025-05-18 15:21:49 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 15:21:49 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 15:21:49 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: AnalysisIndicators.sma() got multiple values for argument 'length'
2025-05-18 16:46:22 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 16:46:22 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 16:46:23 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: AnalysisIndicators.sma() got multiple values for argument 'length'
2025-05-18 16:49:50 - INFO - ERROR - [logger.py:112] - Error in trend detection: /home/b0r3d/Desktop/kursova/.venv2/lib/python3.10/site-packages/talib/_ta_lib.cpython-310-x86_64-linux-gnu.so: undefined symbol: TA_AVGDEV_Lookback
2025-05-18 16:49:50 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 16:49:51 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: /home/b0r3d/Desktop/kursova/.venv2/lib/python3.10/site-packages/talib/_ta_lib.cpython-310-x86_64-linux-gnu.so: undefined symbol: TA_AVGDEV_Lookback
2025-05-18 16:58:01 - INFO - ERROR - [logger.py:112] - Error in trend detection: /home/b0r3d/Desktop/kursova/.venv2/lib/python3.10/site-packages/talib/_ta_lib.cpython-310-x86_64-linux-gnu.so: undefined symbol: TA_AVGDEV_Lookback
2025-05-18 16:58:02 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 16:58:02 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: /home/b0r3d/Desktop/kursova/.venv2/lib/python3.10/site-packages/talib/_ta_lib.cpython-310-x86_64-linux-gnu.so: undefined symbol: TA_AVGDEV_Lookback
2025-05-18 17:02:29 - INFO - ERROR - [logger.py:112] - Error in trend detection: /home/b0r3d/Desktop/kursova/.venv2/lib/python3.10/site-packages/talib/_ta_lib.cpython-310-x86_64-linux-gnu.so: undefined symbol: TA_AVGDEV_Lookback
2025-05-18 17:02:29 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:02:29 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: /home/b0r3d/Desktop/kursova/.venv2/lib/python3.10/site-packages/talib/_ta_lib.cpython-310-x86_64-linux-gnu.so: undefined symbol: TA_AVGDEV_Lookback
2025-05-18 17:03:10 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 17:03:11 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:03:11 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: AnalysisIndicators.sma() got multiple values for argument 'length'
2025-05-18 17:05:16 - INFO - INFO - [logger.py:104] - Running full volatility analysis for ['BTC', 'SOL', 'ETH'] on 1d timeframe
2025-05-18 17:05:16 - INFO - ERROR - [logger.py:112] - Error in full volatility analysis for ['BTC', 'SOL', 'ETH']: 'list' object has no attribute 'upper'
2025-05-18 17:07:05 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 17:07:05 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:07:05 - INFO - ERROR - [logger.py:112] - Error in trend strength calculation: AnalysisIndicators.sma() got multiple values for argument 'length'
2025-05-18 17:08:34 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 17:08:34 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:13:05 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 17:13:05 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:17:06 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 17:17:07 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:24:03 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 17:24:03 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:25:16 - INFO - WARNING - [logger.py:108] - Not enough data points for reliable trend detection
2025-05-18 17:25:16 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:27:47 - INFO - INFO - [logger.py:104] - Identified 2 support and 2 resistance levels
2025-05-18 17:27:48 - INFO - ERROR - [logger.py:112] - Error generating trend summary: 0
